src/components/Person/Person.jsx

```diff
@@ -1 +1,38 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const hasAge = age
+    ? <p className="Person__age">{`I am ${age}`}</p>
+    : null;
```

Comment:
Just put to the HTML and use &&

Reply:
Or you can do like that
```suggestion
  const hasAge = age && <p className="Person__age">{`I am ${age}`}</p>;
```

Reply:
done